modules/fbs-core/web/src/app/page-components/sql-playground/db-control-panel/db-control-co-working/db-control-co-working.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="container">
+  <mat-form-field class="select" appearance="fill">
+    <mat-label>Select Group</mat-label>
+    <mat-select
+      [(value)]="selectedDbGroup"
+      (selectionChange)="activateGroup($event.value)"
```

Comment:
This function no longer exists, but was called here.
```
Error: src/app/page-components/sql-playground/db-control-panel/db-control-co-working/db-control-co-working.component.html:6:26 - error TS2339: Property 'activateGroup' does not exist on type 'DbControlCoWorkingComponent'.
```

Reply:
The missing method has been adjusted.
However, I would like to point out that the issue is partly a click dummy without functionality.